.devcontainer/devcontainer.json

```diff
@@ -47,6 +51,35 @@
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/esm/msg/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/ies/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/util/"
+		],
+		"files.watcherExclude": {
+			"**/.bazel-cache/**": true,
+			"**/.bazel-cache-repo/**": true,
+		},		
+		"bsv.bazel.buildFlags": [
+			"--config=devcontainer",
+		],
+		"bsv.bazel.testFlags": [
+			"--compilation_mode=dbg",
+		],
+		"bsv.bes.enabled": false,
+		"bsv.bzl.codesearch.enabled": false,
+		"bsv.bzl.invocation.buildEventPublishAllActions": false,
+		"bsv.bzl.invocation.enabled": false,
+		"bsv.bzl.invocation.invokeWithBuildEventStreaming": false,
+		"bsv.bzl.lsp.enableCodelensStarlarkDebug": false,
+		"bsv.bzl.lsp.enableCodelensRun": false,
+		"bsv.bzl.remoteCache.enabled": false,
+		"bsv.bzl.starlarkDebugger.enabled": false,
+		"clangd.path": "clangd-12",
+		"clangd.arguments": [
+			"-log=verbose",
+			"-pretty",
+			"--background-index",
+		],
+		"clangd.onConfigChanged": "restart",
+		"bsv.cc.compdb.targets": [
```

Comment:
Does this need to be updated for ... debug symbol support? Or what. If new targets need to be added here down the road lets highlight that in the PR description.

Reply:
I'm not sure if it changes for debug symbol support. (It takes the default build configuration, so it is always debug build by default for our case).

Yes, if we add new targets, we need to edit the configurations. Will add in comments.